app/src/main/java/com/jacobibanez/godot/gpgs/signals/Signals.kt

```diff
@@ -179,16 +178,16 @@ object SnapshotSignals {
      * This signal is emitted when calling the [com.jacobibanez.godot.gpgs.GodotGooglePlayGameServices.loadGame] method
      * or after selecting a saved game in the window opened by the [com.jacobibanez.godot.gpgs.GodotGooglePlayGameServices.showSavedGames] method.
      *
-     * @return A [Dictionary] representing a [com.google.android.gms.games.snapshot.Snapshot](https://developers.google.com/android/reference/com/google/android/gms/games/snapshot/Snapshot).
+     * @return A JSON representing a [com.google.android.gms.games.snapshot.Snapshot](https://developers.google.com/android/reference/com/google/android/gms/games/snapshot/Snapshot).
      */
-    val gameLoaded = SignalInfo("gameLoaded", Dictionary::class.java)
+    val gameLoaded = SignalInfo("gameLoaded", String::class.java)
```

Comment:
I understand you did this for consistency with the other signals, but I was thinking on removing the GSON dependency and using dictionaries and arrays instead, what do you think? this way we can remove a 3rd party library.

Reply:
It makes sense! However, the [Godot docs for 3.5 Android plugins](https://docs.godotengine.org/en/3.5/tutorials/platform/android/android_plugin.html#troubleshooting) states that only specific array types can be used, so I'm unsure if arrays of dictionaries will work, but I'll try.

![image](https://github.com/Iakobs/godot-google-play-game-services-android-plugin/assets/19820741/d5b039bd-fd54-4817-b8be-78f76bcdfe2b)

Reply:
Yeah, it seems that Godot 3.x does not support emitting other than basic types arrays, as the documentation stated. The value becomes `null`.

```
E 0:04:24.886   emit_signal: Error calling method from signal 'eventsLoaded': 'Node(singleton.gd)::_on_events_loaded': Cannot convert argument 1 from Nil to Array..
  <C++ Source>  core/object.cpp:1242 @ emit_signal()
```

Reply:
oooooh! I see!! Let's keep it all JSON strings then, I agree this way is more uniform!